bitnami/kafka/ci/values-with-external-tls.yaml

```
@@ -0,0 +1,11 @@
+# Test values file for generating hidden section of the yaml
+# and check that the rendering is correct
+replicaCount: 3
```

You should also enable external access, right?
```suggestion
replicaCount: 3
externalAccess:
  enabled: false
```